apis/pad-systems/passive-pad-system/api/Apis/DataAnalysisApi.md

```diff
@@ -19,7 +19,7 @@ Analyze biometric capture data for a presentation attack.
 
 |Name | Type | Description  | Notes |
 |------------- | ------------- | ------------- | -------------|
-| **body** | **String**| A biometric sample to be analyzed for evidence of a presentation attack. | |
+| **BiometricSample** | **String**| A biometric sample to be analyzed for evidence of a presentation attack. | |
```

Comment:
Since the content-type here is JSON, should the Type of this parameter be an object instead of a string?  Similar to how it was done in the match-to-id API - https://github.com/TheMdTF/mdtf-public/blob/master/apis/match-to-id/api/Apis/BiometricOperationsApi.md?plain=1#L50